source/CardGames.py

```diff
@@ -279,133 +290,141 @@ def winnnerCheck(self, house, player):
         print(f"You now have ${player_name.money}")
-      else:
-        print("It's a push!")
-        betting_box.collect(winner)
+      elif blackjackOutcome == 3:
+        print("It's a push! You both have blackjack.")
+        print("\nYour Hand: ", player_name.handSum()), player_name.showHand()
+        print("\nDealer's Hand: ", house.handSum(hideSum=True)), house.showHand(showBack= True)
+        betting_box.collect(blackjackOutcome)
         print(f"You now have ${player_name.money}")
+        
+      else:
+        
+        """ Print both hands of the player and dealer """
+        print("\nYour Hand: ", player_name.handSum()), player_name.showHand()
+        print("\nDealer's Hand: ", house.handSum(hideSum=True)), house.showHand(showBack= True)
+
+        """while the player doesn't bust, allow them to choose to hit or pass"""
+        bust = False
+        while not bust :
+          hit_stand = input("\nhit or stand? ")
+          if hit_stand == "hit":
+            dealer.dealCards(1,[player_name])
+            print("\nYour Hand: ", player_name.handSum())  
+            player_name.showHand()
+            if dealer.bustCheck(player_name) == True:
+              print("\nBust!")
+              bust = True
+          else:
+            break
+        
+        """After the player is done, let the house complete its turn"""
+        dealer.dealerHand(house)
+        print("\nDealer's Hand: ", house.handSum())
+        if dealer.bustCheck(house) == True:
+          print("\nBust!")
+        else:
+          house.showHand()
+        
+        """Compare house and player's hands to see who wins. Then handle win/loss interaction."""
+        print("\n")
+        winner = dealer.winnnerCheck(house, player_name)
+        if winner == 2:
+          print("You won!")
+          betting_box.collect(winner)
+          print(f"You now have ${player_name.money}")
+        elif winner == 1:
+          print("You lost. The dealer won!")
+          betting_box.collect(winner)
+          print(f"You now have ${player_name.money}")
+        else:
+          print("It's a push!")
+          betting_box.collect(winner)
+          print(f"You now have ${player_name.money}")
+
+      """"Clear hands and ask if they want to play again"""
+      print("Cards remaining in deck:", deck.size)
+      start = input("\nWould you like to play again (y/n)? ")
+      if start == "n":
+        betting = False
+      else:
+        house.clearHand()
+        player_name.clearHand()
 
-    """"Clear hands and ask if they want to play again"""
-    print("Cards remaining in deck:", deck.size)
-    start = input("\nWould you like to play again (y/n)? ")
-    if start == "n":
-      betting = False
-    else:
-      house.clearHand()
-      player_name.clearHand()
+      if deck.size <=52 and early_shuffle:
+        dealer.resetDeck()
 
-    if deck.size <=52 and early_shuffle:
-      dealer.resetDeck()
\ No newline at end of file
+#blackjack()
```

Comment:
this need to not be commented out for your code to run. Depending on why you commented that out. this might be a more robust solition
 if __name__ == '__main__':
 blackjack

no matter what blackjack needs to be called for this code to be merged

Reply:
When I was running my test it also ran blackjack in the source file. I just commented it out as a quick solution so that wouldn't happen. I'll make sure it runs, thanks!